### PR TITLE
Add flag for disabling schema validation

### PR DESCRIPTION
**Pull request by VVeremjova**
Related to [SDLOPEN-572](https://adc.luxoft.com/jira/browse/SDLOPEN-572)

### CODE REVIEW

**Comment by VVeremjova** [comment]
@LuxoftAKutsan Please review

File: modules/schema_validation.lua [hunk 1]
```diff
@@ -373,7 +373,9 @@ end
 function module.mt.__index:Validate(function_id, function_type, user_data)
   local result = true
   local error_message = {}
-  result, error_message = self:Compare(function_id, function_type, user_data)
+  if config.ValidateSchema then
+    result, error_message = self:Compare(function_id, function_type, user_data)
```
Comment (pestOO) [inline modules/schema_validation.lua]: why not just `return`?

Comment (pestOO) [inline modules/schema_validation.lua]: And after `if end` -> `return true, errorMsgToString({})`?

Comment (pestOO) [inline modules/schema_validation.lua]: Less code, easy to see whats happens.

**Pull request merged**